doc/rbd/nvmeof-target-configure.rst

```diff
@@ -0,0 +1,115 @@
+================================================
+Installing and Configuration the NVMe-oF Targets
+================================================
+
+Traditionally, block-level access to a Ceph storage cluster has been limited to
+QEMU and ``librbd``, which is a key enabler for adoption within OpenStack
```

Comment:
Not quite true -- this disregards the kernel client.  But this is just a nit, I don't insist on rewording this paragraph.

Reply:
It was copied for the iSCSi gateway docs

Reply:
> Not quite true -- this disregards the kernel client. But this is just a nit, I don't insist on rewording this paragraph.

@idryomov, if I add the kernel client to the list of agents that have traditionally enjoyed block-level access, will it make the sentence unambiguously true?

Reply:
I'm thinking of something like:
> Traditionally, block-level access to a Ceph storage cluster has been limited to QEMU and ``librbd``, which is a key enabler for adoption within OpenStack, and the Linux kernel client.

And let's make the same edit in `doc/rbd/iscsi-targets.rst` where this was copied from.

Reply:
> Is the following sentence correct?
> 
> > Traditionally, block-level access to a Ceph storage cluster has been limited to QEMU, `librbd`, and the Linux kernel client.

QEMU accesses Ceph through librbd, not by itself, so not quite correct.  The sentence that I proposed was meant to be parsed as:
> Traditionally, block-level access to a Ceph storage cluster has been limited to:
> 1. QEMU and librbd, which is a key enabler for adoption within OpenStack
> 2. Linux kernel client

I left the "which is a key enabler for adoption within OpenStack" bit in place because it's relevant to QEMU.

Reply:
@idryomov Roger that. I understand now. I'm going to solve this with parentheses (which will likely irritate Anthony D'Atri, to whom I prospectively apologize while nonetheless persisting). Thanks for your clear explanation, Ilya.